src/main/java/utils/configurations/Configurations.java

```diff
@@ -0,0 +1,36 @@
+package utils.configurations;
+
+import com.google.common.collect.ImmutableMap;
+import onlineservices.Service;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import services.WindowControl.WindowControlService;
+import utils.filereaders.CsvFileReader;
+
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+public class Configurations {
+    private final ImmutableMap<String, Service> allServices = ImmutableMap.of(
+            WindowControlService.class.getName(), new WindowControlService()
+    );
+
+    public List<String> getActivatedServicesNames() {
+        CsvFileReader fileReader = new CsvFileReader();
+        return fileReader.readFile(new File(".\\src\\main\\resources\\ActivatedServices.csv"));
+    }
+
+    @Bean(name = "activeServices")
+    public List<Service> getActivatedServices() {
```

Comment:
the name attribute is not needed, you can name the method activatedServices 
and whenever you use:
@Autowire
List<Service> activatedServices;
spring will know that this method will get the activated services